libs/ledger-live-common/src/hooks/recoverFeatueFlag.ts

```diff
@@ -63,9 +63,7 @@ export function useUpsellPath(servicesConfig: Feature<DesktopFeature> | null): s
   return usePath(servicesConfig, uri);
 }
 
-export function useLoginURI(
-  servicesConfig: Feature<MobileFeature | DesktopFeature> | null,
-): string | undefined {
+export function useLoginURI(servicesConfig: Feature<DesktopFeature> | null): string | undefined {
```

Comment:
`account.loginURI` doesn't exist on the feature `protectServicesMobile`. Additionally, this is only use in `Desktop`, so I took the liberty to just keep it for desktop only.